doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -154,3 +272,133 @@ Here is the offline configuration tool workflow:
    :align: center
 
    offline tool workflow
+
+.. _acrn_config_tool_ui:
+
+How to use ACRN config app
+**************************
+ACRN config app is used to read board info, configure and validate scenario settings,
```

Comment:
ACRN config app is a web UI application to read board info, configure and validate scenario setting,